.dev/Manifest.toml

```diff
@@ -1,32 +1,104 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.7.0"
+julia_version = "1.7.2"
```

Comment:
aren't we on 1.7.0 across CliMA?

Reply:
`] up`ing can result in different `julia_version`s, so this tends to switch back-and-forth, depending on who `] up`s and commits manifest changes.

Reply:
so, in general, it shouldn't matter whether some people run projects on 1.7.0 or 1.7.2?

Reply:
Good question, I’m not sure what bug fixes came in 1.71/1.7.2. I’d certainly recommend that everyone updates to the latest version

Reply:
Oh interesting. I have been under the impression that it's really important we use the exact same version of Julia, so I have taken extra care to use 1.7.0 in all my environments, since that's the [current version](https://github.com/CliMA/CalibrateEDMF.jl/blob/main/experiments/SCT2_benchmark/global_parallel/ekp_par_calibration.sbatch#L21) in all the CEDMF pipelines.

Would it be worth updating those slurm scripts to 1.7.2?